vaadin-icons-flow-demo/src/main/resources/META-INF/resources/frontend/all-icons-view.html

```diff
@@ -0,0 +1,155 @@
+<!--
+  ~ Copyright 2000-2017 Vaadin Ltd.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<!-- Dependency resources -->
+
+<!-- Added Web Component dependencies to make Vaadin Designer preview work -->
+<link rel="import" href="../../../frontend/bower_components/polymer/polymer.html">
+<link rel="import" href="../../../frontend/bower_components/iron-meta/iron-meta.html">
+<!--<link rel="import" href="../../../frontend/bower_components/iron-icons/iron-icons.html">-->
+<link rel="import" href="../../../frontend/bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../../../frontend/bower_components/iron-flex-layout/iron-flex-layout.html">
```

Comment:
`iron-flex-layout` is not used, is it needed?

Reply:
removed  imports are unnecessary